README.md

```diff
@@ -26,13 +25,10 @@ Older OCaml versions are unsupported.
 - `COPYING`		  Information about copyright
 - `LGPL`		  Information about LGPL
 - `README.md`		  This file
-- `Makefile.in`	  Makefile used by configure
-- `configure`	  Script generating Makefile
-- `configure.in`	  Script generating configure (with autoconf)
 - `calendar_faq.txt`  FAQ frow which some algorithms come
 - `doc`		  HTML documentation of the API
 - `src`		  Source files directory
-- `target`		  Directory containing the built library
+- `_build/default/`		  Directory containing the built library
```

Comment:
A nit: don't use a tab character here as it's not rendered anyway.

Reply:
it's how it was before…